### PR TITLE
30/skapa sökfunktion

### LINKED ISSUE #30: Skapa en sökfunktion

### PR DESCRIPTION
**Pull request by liinneea-a**
Closes #30 

This PR will
- Create a search screen and search function. 
- Search only based on text input. (Maybe we will implement search by tags or rating etc)

### CODE REVIEW

**Comment by mirrenil** [comment]
> Closes #30
> 
> This PR will
> 
> * Create a search screen and search function.
> * Search only based on text input. (Maybe we will implement search by tags or rating etc)

Would be nice if you could also search for a brand 🤠

**Review by mirrenil — COMMENTED** [review]
Can we change the layout of the search bar so that it matches the figma?

Skärmavbild 2023-01-03 kl  13 22 30

And make sure it's looks good on light mode as well
IMG_2230

**Comment by liinneea-a** [comment]
> > Closes #30
> > This PR will
> > 
> > * Create a search screen and search function.
> > * Search only based on text input. (Maybe we will implement search by tags or rating etc)
> 
> Would be nice if you could also search for a brand 🤠

True! ill add that.

File: screens/SearchScreen.tsx [hunk 1]
```diff
@@ -1,33 +1,157 @@
...
+    borderBottomLeftRadius: 6,
+    alignSelf: "flex-end",
+    marginTop: 10,
+    marginBottom: 10,
+    flexDirection: "row",
+    padding: 10,
+  },
+});
+
+// const card = StyleSheet.create({
```
Comment (mirrenil) [inline screens/SearchScreen.tsx:150]: I guess this can be removed 🐞

**Review by mirrenil — DISMISSED** [review]
Snyggt jobbat!

**Review by mirrenil — APPROVED** [review]
Najs! 💯

**Pull request merged**